src/test/java/core/basesyntax/UserTest.java

```diff
@@ -0,0 +1,105 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class UserTest {
+    private static final String VALID_NAME = "Ivan Zhuravlev";
+    private static final int VALID_AGE = 18;
+    private static final String VALID_PASSWORD = "my password 123";
+    private static final int INVALID_NEGATIVE_AGE = -10;
+    private static final int INVALID_AGE = 15;
+    private static final String INVALID_SHORT_PASSWORD = "12345";
+    private RegistrationServiceImpl registration;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        registration = new RegistrationServiceImpl();
+        user = new User();
+        user.setLogin(VALID_NAME);
+        user.setAge(VALID_AGE);
+        user.setPassword(VALID_PASSWORD);
+    }
+
+    @Test
+    void register_user_ok() {
+        User actual = registration.register(user);
+        assertEquals(user, actual);
+    }
+
+    @Test
+    void register_several_users_ok() {
+        int quantity = 50;
+        for (int i = 0; i < quantity; i++) {
+            user = new User();
+            user.setAge(VALID_AGE);
+            user.setPassword(VALID_PASSWORD);
+            user.setLogin("testName" + i);
+            registration.register(user);
+            assertTrue(Storage.people.contains(user));
+        }
+        assertEquals(quantity, Storage.people.size());
+    }
+
+    @Test
+    void object_user_null_notOk() {
```

Comment:
look at the pattern of test methods naming. Fix it in all places 
![image](https://user-images.githubusercontent.com/32339836/223155607-7a1fc022-3faf-4e5d-a6bd-f176b0ae821e.png)

```suggestion
    void register_userNull_notOk() {
```